adminguide.rst

```diff
@@ -244,27 +244,40 @@ node along with other critical cluster services.
 
 .. _configuration-resource-exclusion:
 
-^^^^^^^^^^^^^^^^^^
-Resource exclusion
-^^^^^^^^^^^^^^^^^^
+^^^^^^^^^^^^^^^^^^^^^^
+Resource configuration
+^^^^^^^^^^^^^^^^^^^^^^
+
+The system resource configuration may be generated in RFC 20 (R version 1)
+form using ``flux R encode``.  At minimum, a hostlist and core idset must
+be specified on the command line, e.g.
+
+.. code-block:: console
+
+ $ flux R encode -H fluke[3,108,6-103] -c0-3 > /etc/flux/system/R
```

Comment:
It might be more illustrative to use long options in the example 
```suggestion
 $ flux R encode --hosts=fluke[3,108,6-103] --cores=0-3 > /etc/flux/system/R
```

Reply:
Good idea!